src/style.css

```diff
@@ -0,0 +1,28 @@
+body {
+  background: #000;
+  margin: 0;
+}
+
+* {
```

Comment:
Try to avoid selector *. Yes in these tasks it doesn't influence performance, but in bigger projects, it can load browser

Reply:
fixed it

Reply:
Why are you return this task? I can't see a comment or mistake